modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -210,12 +210,18 @@ definition {
 	}
 
 	@description = "Verify that the user can add Nodes into diagram by drag&drop: State"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanAddStateNode {
+		property test.name.skip.portal.instance = "ProcessBuilderKaleoDesignerReact#CanAddStateNode";
+
+		Workflow.openWorkflowListView();
 
-		// TODO CanAddStateNode pending implementation
+		Workflow.addNewDefinition(workflowDefinitionTitle = "New Workflow Definition");
 
+		ProcessBuilderKaleoDesignerReact.dragSidebarNodeAndMove(
+			nodeLabel = "State",
+			nodePosition = "50,-420",
+			nodeType = "state");
 	}
```

Comment:
It would be interesting to add a `viewPanelNodePresent` to check the node in the diagram.